src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -1025,6 +1058,47 @@ private static string GetFirstLineSubString(string stringToComplete, out bool ha
             return keyValuePairWithCursor;
         }
 
+        private List<CompletionResult> GetResultForEnum(
+            Type type,
+            CompletionContext completionContext)
+        {
+            var stringToComplete = string.Empty;
+            if (completionContext.TokenAtCursor != null)
+            {
+                stringToComplete = completionContext.TokenAtCursor.Text;
+            }
+
+            var quote = '\'';
+            if (stringToComplete.StartsWith('"'))
+            {
+                quote = '"';
+            }
+
+            var allValues = new List<string>();
+            foreach (var value in Enum.GetValues(type))
```

Comment:
`Enum.GetNames()` allows you to get strings back directly

```suggestion
            foreach (string value in Enum.GetNames(type))
```

Reply:
will change